### DISCUSSION TITLE
blacksheep/background-tasks/

### DISCUSSION THREAD

**Discussion by giscus** [Neoteroi/BlackSheep-Docs#20] [Q&A]
# blacksheep/background-tasks/

The official documentation website of BlackSheep, a fast ASGI web framework for Python asyncio.

https://www.neoteroi.dev/blacksheep/background-tasks/

**Comment by bensarifathi** [comment]
Is there a plan to include celery support for blacksheep ?

**Reply by RobertoPrevato** [reply]
Not for now. Is there anything in BlackSheep that _prevents_ people from using Celery with it?

**Comment by dss010101** [comment]
i dont quite get the purpose of class 'Foo' under 'How to configure background tasks'.  Can you explain a bit more?

**Reply by RobertoPrevato** [reply]
_The following example shows how to configure a background task, **including the activation of a service resolved by the DI container**,_ <--

**Comment by Diman2003** [comment]
celery is good option for backgroud task

**Reply by RobertoPrevato** [reply]
Is there anything in BlackSheep that _prevents_ people from using Celery with it?
